playwright-e2e/dsm/pages/participant-list-page.ts

```diff
@@ -136,20 +138,67 @@ export default class ParticipantListPage {
     return download;
   }
 
-/**
- * Filters the participant list to search for a specific participant when given their guid
- * @param participantGUID the guid of the specific participant to search for
- */
-public async filterListByParticipantGUID(participantGUID: string): Promise<void> {
-  const customizeViewPanel = this.filters.customizeViewPanel;
-  await customizeViewPanel.open();
-  await customizeViewPanel.selectColumns('Participant Columns', ['Participant ID']);
+  /**
+   * Filters the participant list to search for a specific participant when given their guid
+   * @param participantGUID the guid of the specific participant to search for
+   */
+  public async filterListByParticipantGUID(participantGUID: string): Promise<void> {
+    const customizeViewPanel = this.filters.customizeViewPanel;
```

Comment:
indent spaces are wrong. it should be 2 whitespaces instead 4.

Reply:
just to check - do you mean function name? I'm checking on the IDE and it seems the function name is preceded by 2 whitespaces and the content within the function is preceded for 4 whitespaces? I checked how it looks compared to other functions in the class.

Reply:
I meant newline indentation. see this example of too many whitespaces

<img width="484" alt="Screenshot 2023-06-16 at 11 21 49 AM" src="https://github.com/broadinstitute/ddp-angular/assets/35533885/05e9dd51-5dec-4cd4-9a14-4cef8c02dcd2">